### ISSUE TITLE
[Usable - Azure Databases Extension - Stats tab]: Accessible names is not defined for tables present under "Stats & Results" tab.

### ISSUE THREAD

**Issue by chandini1864** [microsoft/vscode-cosmosdb#2669]
### **User Experience:** 

Visually impaired people who depend on screen reader users will not get descriptive information about the multiple tables present in the screen if accessible name is not defined due to which the user will not be able to differentiate and understand the purpose of them. 

**Note:** Login Credentials must not be provided within the bug. 

### **Pre-requisite:** 

Azure database should be installed from extensions in VS code. 

### **Repro Steps** 

1. Open VS Code and navigate to azure databases button present on right pane. 
2. Navigate to anyone of the resource and expand it. Expand a database. Right click on any collection and select "Open Query editor(Preview)". 
3. Query editor screen appears. Run any sample query.  
4. Select stats tab. Navigate through all controls of stats tab. 
5. Verify if Accessible names is not defined for tables present under "Stats & Results" tab. 

### **Actual Result:** 

Accessible names is not defined for tables present under "Stats & Results" tab i.e while navigating with screen reader Narrator in table mode it announces for 1st table as “Table with 15 rows 2 columns” and for the 2nd table as ”Table with 5 rows and 2 columns”.

### **Expected Result:** 

Accessible names should be defined for tables present under "Stats & Results" tab i.e for table name should be defined as “Query metrics” and for second table it should be defined as “Index Utilization Information”.

### **Reference links:** 
 
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries. 

### **Test Environment:**   

- OS Version: 24H2 (OS Build 26100.3476) 
- Visual Studio Code Version: 1.99.0 (user setup) 
- Tool: Accessibility Insights for Win32 
- Azure Databases for VS Code (Preview) Version : 0.25.2 
- Matrix:Win32+Narrator

<img width="1471" alt="Image" src="https://github.com/user-attachments/assets/53fb7c66-ec10-44de-946f-f5787a0e4fa8" />

**Comment by ChandiniSmiley** [comment]
#A11yUsable;#Accessibility;#Benchmark;# GH-HCL-AzureDatabasesExtension-Apr2025;#HCL;#Win32;

**Comment by chandini1864** [comment]
As per in the group conversation, fixes will be provided by the end of July.
So as per the process adding the pending environment and adding July 2025 ETA tag and assigning back to the product team please keep the state as resolved until the fix been deployed. Kindly assign back to us when changes are deployed.

**Comment by sevoku** [comment]
@chandini1864 the fix won't be deployed until it's validated. Please validate it using the latest build from main: https://github.com/microsoft/vscode-cosmosdb/actions/runs/16441560689/job/46463438375
